zipline/pipeline/data/dataset.py

```diff
@@ -267,6 +304,13 @@ def currency_conversion(self):
         """
         return self._currency_conversion
 
+    @property
+    def currency_aware(self):
+        """
+        Whether or not this column produces currency-denominated data.
+        """
+        return self._currency_aware
```

Comment:
Are there clients that need to be able to access this attribute?

Reply:
Yes I was using this downstream.